common/lib/driver-definitions/src/driverError.ts

```diff
@@ -51,10 +51,10 @@ export enum DriverErrorType {
     writeError = "writeError",
 
     /**
-     * Generic fetch failure.
-     * Most of such failures are due to client being offline, or DNS is not reachable, such errors map to
-     * DriverErrorType.offlineError. Anything else that can't be diagnose as likely offline maps to this error.
-     * This can also indicate no response from server.
+     * A generic fetch failure that indicates we were not able to get a response from the server.
+     * This may be due to the client being offline (though, if we are able to detect offline state it will be
```

Comment:
Might be worth mentioning that the offline detection has false negatives - sometimes it says online when it is offline.  So we truly do expect some offline cases to come through as fetchFailure.

Maybe also update the comment for offlineError to indicate somehow it's a best guess?

Reply:
My read of the comments is that they already say this (offlineError comment indicates that it signifies being offline, and the fetchFailure comment acknowledges that it might arise from being offline if we can't detect the offline state) - is there some wording you were looking for?